extensions/wikia/HAWelcome/HAWelcomeHooks.php

```diff
@@ -23,186 +23,6 @@ public static function queueHAWelcomeTask( $wgCityId, $oTitle, $aParams ) {
 	}
 
 	/**
```

Comment:
The below code was removed because it was superseded by a new hook handler (see https://github.com/Wikia/app/pull/8042/files#diff-7d403cb912d0576b398c56344730ae26L51)